pkg/controller/component/reclaimer/pvc_reclaimer.go

```diff
@@ -58,41 +58,56 @@ func (p *pvcReclaimer) reclaimPV(nc *v1alpha1.NebulaCluster) error {
 		pvc := pvcs[i]
 		pvcName := pvc.GetName()
 		if !label.Label(pvc.Labels).IsNebulaComponent() {
-			log.V(4).Info("pvc of cluster skip reclaim for not nebula component", "pvcName", pvcName)
+			log.V(4).Info("skip reclaim for not nebula component", "pvcName", pvcName)
 			continue
 		}
 
 		if pvc.Status.Phase != corev1.ClaimBound {
-			log.V(4).Info("pvc of cluster skip reclaim for pvc status is not bound", "pvcName", pvcName)
+			log.V(4).Info("skip reclaim for pvc status is not bound", "pvcName", pvcName)
 			continue
 		}
 
 		if pvc.DeletionTimestamp != nil {
-			log.V(4).Info("pvc of cluster skip reclaim for pvc has been deleted", "pvcName", pvcName)
+			log.V(4).Info("skip reclaim for pvc has been deleted", "pvcName", pvcName)
 			continue
 		}
 
 		if pvc.Annotations[annotation.AnnPVCDeferDeletingKey] == "" {
-			log.V(4).Info("pvc of cluster skip reclaim for pvc has not been marked as defer deleting pvc", "pvcName", pvcName)
+			log.V(4).Info("skip reclaim for pvc has not been marked as defer deleting pvc", "pvcName", pvcName)
 			continue
 		}
 
```

Comment:
if pvc.Annotations' AnnPvReclaimKey == "false" , does it need skip too ?, if not skip delete pvc , spec. EnablePVReclaim maybe become confused

Reply:
pvc gc and pvc relaimer are seprated logic, if the nebula cluster deleted,  pvc gc will work, if the nebula component scale in, pvc reclaimer will work.